app/jsontableschema/constraints.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
```

Comment:
@pwalsh I need to use jsontableschema, but can't use ES6 version in the project, while it is run on NodeJS 4.x. I did automatic cloning of repository and building the scripts, but found that it would be better just to add it to the code base of the project.